app/models/Signup.js

```diff
@@ -129,8 +131,10 @@ signupSchema.statics.lookupCurrent = function (user, campaign) {
       .then(signupDoc => resolve(signupDoc))
       .catch((err) => {
         app.locals.stathatError(statName, err);
```

Comment:
should we send this new custom error message to stathat too? It could probably help troubleshoot in the future.